packages/packagers/js/src/ScopeHoistingPackager.js

```diff
@@ -676,6 +676,10 @@ ${code}
       exportSymbol,
       symbol,
     } = this.bundleGraph.resolveSymbol(resolved, imported, this.bundle);
+    if (resolvedAsset.type !== 'js') {
+      // Graceful fallback for non-js imports
+      return '{}';
+    }
```

Comment:
The non-scope-hoisting packager creates one new asset per non-JS dependency resolution, while this currently returns a new `{}` for every single use of that dependency... :

```js
const x = require("./index.css");
console.log(x);
console.log(x);
```
becomes
```js
console.log({});
console.log({});
```

Reply:
An alternative would be to insert this asset in the jsruntime, but that would require calling `getDependencyResolution` on every single dependency to find the non-js ones.

Reply:
This seems fine to me. Not sure if anyone will really care that they get different objects back rather than the same one. Pretty edge case.